mne/io/base.py

```diff
@@ -688,12 +693,21 @@ def apply_hilbert(self, picks, envelope=False, n_jobs=1, verbose=None):
         MNE inverse solution, the enevlope in source space can be obtained
         by computing the analytic signal in sensor space, applying the MNE
         inverse, and computing the envelope in source space.
+
+        Also note that the n_fft parameter will allow you to pad the signal
```

Comment:
but actually it shouldn't matter if your padding is symmetrical or not, since the FFT is circular, so never mind!